packages/react-cards/src/components/Card/CardPage.tsx

```diff
@@ -15,8 +15,8 @@ export class CardPage extends React.Component<IComponentDemoPageProps, {}> {
         componentName="Card"
         exampleCards={
           <div>
-            <ExampleCard title="Basic Card" code={CardBasicExampleCode}>
-              <CardBasicExample />
+            <ExampleCard title="Vertical Card" code={CardVerticalExampleCode}>
+              <CardVerticalExample />
```

Comment:
With regards to Card documentation, the first questions I had revolved around CardItem vs. CardSection, whether or not they are interchangeable (which is yes, but by looking at code), and whether or not they have to be nested when used.